android/src/main/java/com/tailscale/ipn/ui/viewModel/IpnViewModel.kt

```diff
@@ -147,7 +155,7 @@ open class IpnViewModel : ViewModel() {
 
   fun deleteProfile(profile: IpnLocal.LoginProfile, completionHandler: (Result<String>) -> Unit) {
     Client(viewModelScope).deleteProfile(profile) {
-      viewModelScope.launch { loadUserProfiles() }
+      launchForInit { loadUserProfiles() }
```

Comment:
would we want to use launchForInit in addProfile and switchProfile too?

Reply:
I've since removed this, turns out there was a better solution.